src/collections/repositories/collection.repository.ts

```diff
@@ -12,30 +13,106 @@ export class CollectionRepository extends Repository<Collection> {
     super(Collection, dataSource.createEntityManager());
   }
 
-  async pagination(paginationDto: PaginationDto): Promise<Collection[] | undefined> {
-    const { limit, skip } = paginationDto;
+  async getNewest(qty: number): Promise<{ totalCount: number; collections: Collection[] }> {
+    const queryOptions = {
+      relations: {
+        owner: true,
+        image: true,
+      },
+      select: {
+        owner: {
+          address: true,
+        },
+      },
+      take: qty,
+    } as unknown as FindManyOptions<Collection>;
 
-    const data = await this.find({
-      relations: { items: true },
+    const [collections, totalCount] = await this.findAndCount(queryOptions);
+
+    return {
+      totalCount,
+      collections,
+    };
+  }
+
+  async pagination(
+    paginationDto: CollectionPaginationDto
+  ): Promise<CollectionResponseDto | undefined> {
+    const { limit, skip, creationOrder, mintType, popularityOrder } = paginationDto;
```

Comment:
Check this warning: minType is never used.

Reply:
Yep, its ready for when the lazy minting feature is done